src/board/system76/galp6/board.mk

```diff
@@ -5,8 +5,8 @@ EC=it5570e
 # Enable eSPI
 CFLAGS+=-DEC_ESPI=1
 
-# Apply PMC hack for S0ix
-CFLAGS+=-DPMC_S0IX_HACK=1
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
```

Comment:
As above, this device will not be using S0ix anymore.

Reply:
Done.

Reply:
There was an error when pushing which i didn't notice, should be fine now.